nautobot/utilities/testing/views.py

```diff
@@ -447,7 +447,7 @@ def test_create_object_with_constrained_permission(self):
             # Assign constrained permission
             obj_perm = ObjectPermission(
                 name="Test permission",
-                constraints={"pk": str(uuid.uuid4())},  # Dummy permission to deny all
+                constraints={"pk": str(uuid.uuid4())},  # Example permission to deny all
```

Comment:
I'd maybe change the comment throughout this file to something like  `# Match a non-existent pk (i.e., deny all)`